src/components/admin/__tests__/HearingToolbar.test.jsx

```
@@ -0,0 +1,156 @@
+import React from 'react';
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)